### PR TITLE
InputDialog: pre-populate input text

### LINKED ISSUE #2567: No pre-populated text when editing pre-existing description, account label or address label
Current text is not pre-populated when you are editing a pre-existing description (Transactions page), an account label (Account page) or an address label (Receive page).

descrip

### PR DESCRIPTION
**Pull request by selsta**
Fixes #2567

### CODE REVIEW

**Comment by selsta** [comment]
Will update PR later, current patch is wrong.

**Comment by selsta** [comment]
That’s what I had in the beginning, I used this variant because the diff is smaller. Both is fine for me.

**Comment by xiphon** [comment]
> That’s what I had in the beginning, I used this variant because the diff is smaller. Both is fine for me.

Great, would be nice to use the optional argument.

**Comment by selsta** [comment]
updated

File: pages/Account.qml [hunk 1]
```diff
@@ -354,7 +353,6 @@ Rectangle {
                 text: qsTr("Create new account") + translationManager.emptyString; 
                 onClicked: { 
                     inputDialog.labelText = qsTr("Set the label of the new account:") + translationManager.emptyString
-                    inputDialog.inputText = qsTr("(Untitled)") + translationManager.emptyString
```
Comment (xiphon) [inline pages/Account.qml]: Did you forget this one?

Comment (xiphon) [reply pages/Account.qml]: Ah, seems it is intentional

**Pull request merged**